.air.toml

```
@@ -0,0 +1,32 @@
+root = "."
+tmp_dir = "tmp"
```

This creates `tmp` directory as not staged file, if `make` fails.

```suggestion
tmp_dir = ".tmp"
```